product/views/ManageIQ_Providers_PhysicalInfraManager.yaml

```diff
@@ -0,0 +1,98 @@
+#
+# This is an MIQ Report configuration file
+#   Single value parameters are specified as:
+#     single_value_parm: value
+#   Multiple value parameters are specified as:
+#     multi_value_parm:
+#       - value 1
+#       - value 2
+#
+
+# Report title
+title: Physical Infrastructure Providers

```

Comment:
Actually this whole file on the github preview is riddled with `^M`s must have been an editor faux pas.